### ISSUE TITLE
when joining wifi, should have a list of prioritized list of wifi to join

### ISSUE THREAD

**Issue by alberttwong** [resin-os/resinos#222]
My problem is that flashing for each wifi network or network configuration is a pain in the ass.   I'd like to be in a prioritized list of wifi entries that the device tries to join.

as a device, I want to join a wifi network and gain internet connectivity.   If my primary wifi network isn't available, I want to go down the list of wifi access points that has been configured for me and try them until I am able to connect.

**Comment by agherzan** [comment]
Hi @alberttwong . Thanks for your feature request. We will discuss it internally and update the issue accordingly. CC @shaunmulligan

**Comment by floion** [comment]
I think having the option to define multiple network configs is currently being discussed. Confirm @shaunmulligan ?

**Comment by shaunmulligan** [comment]
@floion yes that is correct, its on our roadmap, but no current ETA on it.
